addons/folders/userscript.js

```diff
@@ -1218,6 +1218,19 @@ export default async function ({ addon, console, msg }) {
         newIndex
       );
     };
+
+    // Temporal bug fix for #5762
+    const originalShareSoundToTarget = vm.shareSoundToTarget;
+    vm.shareSoundToTarget = function (...args) {
+      const target = this.runtime.getTargetById(args[1]);
+      if (!target) {
+        // Avoid reading property from null
+        return Promise.reject();
```

Comment:
Since it's going to end up logged as a scary red warning in the JS console, let's be a bit more specific than `Uncaught (in promise) undefined`
```suggestion
        return Promise.reject(new Error("Dropping sound into folder is not supported"));
```